packages/hoppscotch-sh-admin/src/helpers/configs.ts

```diff
@@ -55,12 +55,23 @@ export type UpdatedConfigs = {
   value: string;
 };
 
+export type ConfigTransform = {
+  config: Config[];
+  enabled?: boolean;
+  field?: Record<string, string | boolean> | string;
```

Comment:
We could rename this to `fields` since it is essentially the list of fields applicable to the provider.